src/app.js

```diff
@@ -95,30 +130,107 @@ io.use((socket, next) => {
 
 //------------------------------------------------------------------
 
-fastify.get('/viewer/rs/studies', (req, reply) => {
-  emitToWsClient(reply, 'STUDY', req.query);
+fastify.get('/viewer/rs/studies', (req, reply) => (
+  emitToWsClient(reply, 'STUDY', req.query)
+));
+
+//------------------------------------------------------------------
+
+fastify.get('/viewer/rs/studies/:studyInstanceUid', async (req, reply) => {
+  const { query } = req;
+  query.studyInstanceUid = req.params.studyInstanceUid;
```

Comment:
does lint not whinge here?

I think I'd prefer: 
```
const query = {
  ...req.query,
  studyInstanceUid: req.params.studyInstanceUid
}
```

edit: tho I do now see this is like this elsewhere already

Reply:
Linting wasn't enabled when I was on this branch - I enabled it later :)

Reply:
Also, that format above gives "Parsing error: Unexpected token ..." - I think it's because internally it's 'any' type, which may not be spreadable. Will see if I can change it later branches when I have enabled TS.